hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java

```diff
@@ -157,6 +156,7 @@
 
 import org.apache.hadoop.hbase.shaded.protobuf.ProtobufUtil;
 
+
```

Comment:
Nit: remove useless empty line.

Reply:
Done.

Reply:
Let me address review in a sub-issue. I want to get a test run in to make sure nothing broke.